README.md

```diff
@@ -1,29 +1,71 @@
-# CNTK Object Recognition Video Tagging Tool
+# CNTK Object Detection Video Tagging Tool
 
-This tool provides end to end support for generating datasets for and validating Object Recognition Models with CNTK.
+This tool provides end to end support for generating datasets for and validating Object Detection Models with CNTK.
 
 It supports the following scenarios:
 
 - Computer Assisted Tagging of Objects In Video Using Custom implementation of Camshift Algorithm
-- Export Tags to CNTK format for training a CNTK object recognition model.
+- Exporting Tags to CNTK format for training a CNTK object recognition model.
 - Running and validating a trained CNTK object recognition model on new videos to generate stronger models. (Windows only for now, GPU recommended)
 
-## To Use:
- 1. Download the release binary.
- 2. Extract and run the app
- 3. Load a video.
- 4. Configure the tagging job specifying the following preferences
+## Table of Contents
+ - [Prerequisites](#prerequisites)
+ - [Tagging Job](#taggingjob)
+ - [Review/Itterate CNTK Object Detction Model](#review)
+ - [Upcoming Features](#upcoming)
+ 
+<a name="prerequisites"></a>
+## Prerequisites
+ 1. Install [CNTK and Dependencies](https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#setup)
+ 2. Download and Extract the app [release package](https://github.com/CatalystCode/CNTK-Object-Detection-Video-Tagging-Tool/releases)
+ 3. Configure CNTK-Config.json with the following properties
+ 
+  ```json
+   {
+    "cntkPath" : "{CNTK Path default is c:/local/cntk}", 
+    "cntkEnv" : "{CNTK Enviorment default is cntk-py34}" 
+   }
+  ```
+ 4. Run the app
+ 
+ <img src="/media/1_home.jpg" alt="Home Page" height="300px" width="500px"/>
```

Comment:
Hmmm..I'm not sure if you have to show this screenshot, perhaps just showing the "open file" window and saying that you can drag & drop is enough